docs/releases/2.0.0b5.rst

```diff
@@ -0,0 +1,234 @@
+Translate Toolkit 2.0.0b5
+*************************
+
+*Released on 29 August 2016*
+
+This release contains many improvements and bug fixes. While it contains many
+general improvements, it also specifically contains needed changes and
+optimizations for the upcoming `Pootle <http://pootle.translatehouse.org/>`_
+2.8.0 and `Virtaal <http://virtaal.translatehouse.org>`_ releases.
```

Comment:
Can we try liven up this first paragraph.  Its the first thing anyone sees and its the same boring text very single release.